kubernetes-maven-plugin/doc/src/main/asciidoc/inc/_faq.adoc

```diff
@@ -141,3 +141,21 @@ You can also provide a host for it in XML config like this:
 
 You can find an example in our link:https://github.com/eclipse/jkube/tree/master/quickstarts/maven/spring-boot[spring-boot]
 quickstart in `kubernetes-with-ingress` profile.
+
+=== How to configure image name generated by Eclipse JKube?
+
+If you want to configure image name generated by Eclipse JKube which is `%g/%a:%l` by default(see <<image-name>>). It will depend upon what mode you're using in Eclipse JKube:
+
+- If you're using zero configuration mode, which means you depend on Eclipse JKube <<generators>> to generate an opinionated image, you might be able to do it using `jkube.generator.name` flag.
```

Comment:
```suggestion
- If you're using zero configuration mode, which means you depend on Eclipse JKube <<generators>> to generate an opinionated image, you will be able to do it using `jkube.generator.name` maven property.
```

Reply:
Maybe also link the `zero configuration mode` words to the corresponding section in the documentation